new/LIT5905FidalaHawarya.xml

```diff
@@ -0,0 +1,72 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5905FidalaHawarya" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ፊደለ፡ ሐዋርያ፡</title>
+                <title corresp="#t1" type="normalized">Fidala ḥawāryā</title>
+                <title xml:id="t2" xml:lang="en">Apostle's Alphabet</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>First seven or eight verses of the
+                  <ref corresp="LIT3509Epistle" type="work">First Epistle of John</ref> used in the traditional
+                schools for training of reading.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Bible"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="DidacticMaterial"></term>
+                    <term key="NewTestament"/>
+                    <term key="Translation"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2020-04-29">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+              <listBibl type="secondary">
+                <bibl>
+                  <ptr target="bm:Platonov1996Katalog"/>
+                  <citedRange>26-29</citedRange>
+                </bibl>
+              </listBibl>
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT5905FidalaHawarya" passive="LIT3509Epistle"/>
```

Comment:
saws:formsPartOf is used to declare that a defined Textual Unit is always part of another one.
I am not sure this is applicable here.
I would better put saws:isRelatedTo and add a desc element to explain the relation